UnitsNet/CustomCode/Quantities/Duration.extra.cs

```diff
@@ -97,5 +97,11 @@ public TimeSpan ToTimeSpan()
         {
             return Volume.FromCubicMeters(volumeFlow.CubicMetersPerSecond * duration.Seconds);
         }
+
+        /// <summary>Calculate <see cref="ElectricCharge"/> from <see cref="Duration"/> multiplied by <see cref="ElectricCurrent"/>.</summary>
+        public static ElectricCharge operator *(Duration time, ElectricCurrent current)
```

Comment:
For operator*, both orders should normally be provided. Add `operator *(ElectricCurrent, Duration)` as well.

Reply:
It is provided in ElectricCurrent.extra.cs, as the wiki says the methods should live in the same class as the first parameter.

Reply:
My bad, sorry.